examples/PostgreSql/PostgreSql.Tests.ps1

```diff
@@ -0,0 +1,70 @@
+﻿BeforeAll {
+  Import-Module Az.InfrastructureTesting
+
+  $Script:rgName       = 'rg-test'
+  $Script:location     = 'westus3'
+  $Script:serverName   = 'samplepostgresqlserver'
+  $Script:noServerName = 'nosamplepostgresqlserver'
```

Comment:
Can you move the no server name variable to a before all in the describe block so it is consistent with our other examples?

Reply:
I can, but I actually made a conscious decision to place it here. My reasoning was that there is only a single `Describe` block, so moving only that one variable into the `Describe.BeforeAll` seemed odd to me. The others that do it, define at the root `BeforeAll` variables that are used across all `Describe` blocks, and then at the `Describe.BeforeAll` for those that are only used within the specific `Describe` block. Given that there is only a single `Describe` block, the options I thought that made the most sense were to either put all of the variables into the root `BeforeAll` or to put them all into `Describe.BeforeAll`. In the end I chose the former because it is more consistent with the other tests putting the variables that are used by all `Describe` blocks (in this case the only `Describe` block) into the root `BeforeAll`.

If you feel strongly about it, we can certainly discuss it further.

Reply:
We have a number of other examples that have only a single `Describe` block. I agree with you that it makes more sense to move all variables to the root `BeforeAll` in this case. But since we are trying to maintain consistency across our codebase, we should update all those examples as well.

Reply:
Good call! Done!